lib/policy/checker/checker.go

```diff
@@ -19,15 +19,12 @@ type List []Impl
 func (l List) Check(r *http.Request) (bool, error) {
 	for _, c := range l {
 		ok, err := c.Check(r)
-		if err != nil {
+		if err != nil || !ok {
 			return ok, err
 		}
-		if ok {
-			return ok, nil
-		}
 	}
 
-	return false, nil
+	return true, nil
```

Comment:
This change was made to ensure that we don't try to do any unnecessary reverse DNS checks if other checkers (such as user agent or CEL) fail. I assumed this was already the case and I am not sure if changing this will have unintended consequences.